packages/near-membrane-base/src/environment.ts

```diff
@@ -142,7 +136,12 @@ export class VirtualEnvironment implements MembraneBroker {
 
                 if (typeof blueDescriptor.get === 'function') {
                     const { get: blueGetter } = blueDescriptor;
-                    const blueDistortedGetter: () => BlueValue = WeakMapGet(this.distortionMap, blueGetter) || blueGetter;
+                    // Note: the reason why we don't use broker.getRedValue here is because we want that proxy to be
+                    // lazy. This brings other questions: what about error control? Do we have test for this?
+                    // Can we optimize this so after the first call we don't pay the cost of wrapping anymore?
+                    // TODO: isn't easier just to not do any lazy stuff anymore considering that the creation of those
+                    // proxies is now faster?
+                    const blueDistortedGetter: () => BlueValue = this.distortionCallback(blueGetter) as () => BlueValue;
```

Comment:
```js
BlueValue = this.distortionCallback(blueGetter) as () => BlueValue
```
This is specifying a type if `BlueValue` and not assigning it to a var named `BlueValue` right?